profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/WindowsThreadInfo.cpp

```diff
@@ -26,3 +26,15 @@ HANDLE WindowsThreadInfo::GetOsThreadHandle() const
 {
     return _handle;
 }
+
+std::string WindowsThreadInfo::GetProfileThreadId()
```

Comment:
not sur to understand why this function (and the other ) is not supposed to be called on Windows.

Reply:
I need to "inject" thread id and name when receiving Framework events instead of those retrieved synchronously.
So, I added the 2 methods that are only implemented for FrameworkThreadInfo but not in (Windows/Linux)ThreadInfo